paasta_tools/paasta_metastatus.py

```diff
@@ -76,59 +74,73 @@ def parse_args(argv):
     return parser.parse_args(argv)
 
 
+def verify_marathon_clients_and_return_framework_ids(marathon_clients):
```

Comment:
Can you add a docstring to this? I'm not understanding what it's doing, especially w/r/t the second to last line (`marathon_clients[:] = ...`).

Reply:
I wanted to implement something more intelligent than failing when one marathon shard is down, but I think I abandon this idea this time, because I don't feel like I have a good solution.